packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -106,9 +108,11 @@ class Plugin implements CLNPlugin {
         .where((rpc) => rpc.name != "init" && rpc.name != "getmanifest")
         .map((rpc) => rpc.toMap())
         .toList();
-    response["subscriptions"] = plugin.subscriptions;
     response["hooks"] = plugin.hooks.toList();
-    response["notifications"] = [];
+    response["subscriptions"] = plugin.subscriptions.values
+        .map((subscription) => subscription.toMap())
```

Comment:
```suggestion
        .map((subscription) => subscription.name)
```

if you read the docs, the subscription is a list of string, and not a list of object, the toMap is useful for the notification, so lets use the name propriety here.

I think this is a copy and paste refused, so for this reason I'm forcing you to write test, if you are difficulties to develop a meaninful test I will write it